docs/DeveloperGuide.adoc

```diff
@@ -787,8 +787,20 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 * can type fast
 * prefers typing over mouse input
 * is reasonably comfortable using CLI apps
+* has a need to manage finances with fellow contacts
+* has a need to keep track of personal balance
+* has a need to keep track of transactions between contacts
+* travels with a group and lends money/borrows money.
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*:
+
+* manage contacts faster than a typical mouse/GUI driven app
+* easily keep track of balances and debts between contacts
+* easily see your overall balance
+* easily see contacts that owe you money
+* easily see contacts to which you are in debt
+* easily keep track of transactions between contacts
+* easily update balance once debts are paid
```

Comment:
Don't confuse features and value proposition, these points seems like features with a word "easily".